commit 191d3d74cce971c4d5b2cd40c8039549846ba457
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sat Jul 4 20:37:03 2020 +0200

    Fix ucs2keysym line for Euro sign
    
    AFAICT, 0x13a4 has never been the keysym for the Euro sign.
    Even since X11R6.6 we had #define XK_EuroSign 0x20ac.
    
    This fixes synthesizing it.

diff --git a/registryd/ucs2keysym.c b/registryd/ucs2keysym.c
index b4967be..ad237c0 100644
--- a/registryd/ucs2keysym.c
+++ b/registryd/ucs2keysym.c
@@ -549,7 +549,7 @@ struct codepair {
   { 0x0afc, 0x2038 }, /*                       caret ‸ CARET */
   { 0x047e, 0x203e }, /*                    overline ‾ OVERLINE */
   { 0x0eff, 0x20a9 }, /*                  Korean_Won ₩ WON SIGN */
-  { 0x13a4, 0x20ac }, /*                        Euro € EURO SIGN */
+  { 0x20ac, 0x20ac }, /*                        Euro € EURO SIGN */
 
   { 0x0ab8, 0x2105 }, /*                      careof ℅ CARE OF */
   { 0x06b0, 0x2116 }, /*                  numerosign № NUMERO SIGN */
